roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -9,10 +9,11 @@ openshift_logging_mux_namespace: logging
 
 ### Common settings
 openshift_logging_mux_nodeselector: "{{ openshift_hosted_logging_mux_nodeselector_label | default('') | map_from_pairs }}"
-openshift_logging_mux_cpu_limit: 500m
-openshift_logging_mux_memory_limit: 2Gi
-openshift_logging_mux_buffer_queue_limit: 1024
-openshift_logging_mux_buffer_size_limit: 1m
+openshift_logging_mux_cpu_limit: null
+openshift_logging_mux_cpu_request: 100m
+openshift_logging_mux_memory_limit: 512Mi
```

Comment:
Note here that we are lowering the mux memory limit to some more reasonable as a default, 512 MB, and adjust the `buffer_queue_limit`/`buffer_chunk_size` parameters to use half of that memory.  It is not clear we needed 2GB of memory as a default.